guide/C/ch_reports.xml

```diff
@@ -1,10 +1,11 @@
 ﻿<!--
       (Do not remove this comment block.)
   Version: 2.6.15
```

Comment:
Do not forget to update the version number at least to the minimal required one
* in both files
* when everything else is OK